README.md

```diff
@@ -1,52 +1,50 @@
 # Amazon Sumerian Hosts
 
-Amazon Sumerian Hosts is an experimental open source project that aims to make it easy to create interactive animated 3D characters that can be rendered on the Web and leverage AWS Services such as [Amazon Polly](https://aws.amazon.com/polly/). 
-It defines a Javascript API for managing animations, synthesizing and playing speech with Amazon Polly, and generating lipsync animation at runtime in sync with Polly generated audio. Amazon Sumerian Hosts is divided up into three packages:
+Amazon Sumerian Hosts is an experimental open source project that aims to make it easy to create interactive animated 3D characters that can be rendered on the Web and leverage AWS Services such as [Amazon Polly](https://aws.amazon.com/polly/) and [Amazon Lex](https://aws.amazon.com/lex/). 
+It defines a Javascript API for managing animations, synthesizing and playing speech with Amazon Polly, generating lipsync animation at runtime in sync with Polly generated audio and interacting with Amazon Lex bot. Amazon Sumerian Hosts is divided up into four packages:
 * [Amazon-Sumerian-Hosts-Core](packages/amazon-sumerian-hosts-core) is the core API that can be extended to support the Web rendering engine of your choice

```

Comment:
nit: I'd suggest either listing these using NPM package notation (eg. `@amazon-sumerian-hosts/core`) or with the full pack using correct case (`packages/amazon-sumerian-hosts-core`). I worry it may be confusing to introduce a 3rd naming style.

Reply:
Renamed these